python_modules/dagster/dagster/_core/definitions/assets_job.py

```diff
@@ -155,70 +146,38 @@ def asset2(asset1):
     """
     from dagster._core.execution.build_resources import wrap_resources_for_execution
 
-    check.str_param(name, "name")
```

Comment:
This codeblock forced me to switch to graphite to find an appropriate meme.

Reply:
/[![kanyewow](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/WjlTemxb6oA4PgZFmj08/4c4a23b5-781b-4244-99db-a780a01f1d64/kanyewow.gif)](https://app.graphite.dev/settings/meme-library?org=dagster-io)